Makefile

```diff
@@ -129,3 +169,21 @@ wo-deadletter:
 		datacube-alchemist run-from-queue --config-file ./examples/c3_config_wo.yaml \
 		--queue dea-dev-eks-alchemist-c3-processing-wo-deadletter \
 		--queue-timeout=1200
+
+product_nci:
+	docker-compose exec alchemist \
+		datacube product add \
+        https://raw.githubusercontent.com/GeoscienceAustralia/digitalearthau/develop/digitalearthau/config/eo3/products/ard_ls5.odc-product.yaml \
+        https://raw.githubusercontent.com/GeoscienceAustralia/digitalearthau/develop/digitalearthau/config/eo3/products/ard_ls7.odc-product.yaml \
+        https://raw.githubusercontent.com/GeoscienceAustralia/digitalearthau/develop/digitalearthau/config/eo3/products/ard_ls8.odc-product.yaml
+
+index_nci:
+	docker-compose exec alchemist \
+		bash -c "\
+			find /code/examples/ga_ls8c_ard_3/ -name "*.odc-metadata.yaml" | \
+			xargs datacube dataset add \
+			--ignore-lineage --confirm-ignore-lineage --product=ga_ls8c_ard_3 \
+		"
+
+process_nci:
+	echo "Fake"
```

Comment:
Sorry, this is my bad. Can we please remove this?

Reply:
Removed.